includes/admin/class-wc-rest-payments-settings-controller.php

```diff
@@ -101,12 +113,17 @@ function ( WC_Payment_Gateway_WCPay $gateway ) {
 
 		$enabled_payment_method_ids = wp_list_pluck( $enabled_payment_methods, 'id' );
 
-		return new WP_REST_Response(
-			[
-				'enabled_payment_method_ids' => array_values( $enabled_payment_method_ids ),
-				'is_wcpay_enabled'           => $this->wcpay_gateway->is_enabled(),
-			]
-		);
+		$response = [
+			'enabled_payment_method_ids' => array_values( $enabled_payment_method_ids ),
+			'is_wcpay_enabled'           => $this->wcpay_gateway->is_enabled(),
+		];
+
+		if ( WC_Payments_Features::is_grouped_settings_enabled() ) {
+			$response['is_digital_wallets_enabled']       = $this->digital_wallets_gateway->is_enabled();
+			$response['digital_wallets_enabled_sections'] = $this->digital_wallets_gateway->get_option( 'digital_wallets_enabled_sections' );
+		}
```

Comment:
As mentioned [above](https://github.com/Automattic/woocommerce-payments/pull/1810/files#r635565036), We probably will get rid of the `Digital_Wallets_Sections` class. We can probably change this to retrieve options directly using `$this->wcpay_gateway`.